docs/spec/resources.md

```diff
@@ -76,7 +111,35 @@ A nested resource may access properties of its parent resource. Other resources
 
 More info on the nested child resource access operator can be found in the [expressions spec](./expressions.md#nested-resource-accessors).
 
-**Note:** Alternatively, you can use the `parent` property on child resources to declare a child resource as a top-level resource. [Read more about the "parent" property](./resource-scopes.md#'parent'-property-syntax)
+### Manually building the name property
+Another option is to supply all of the hierarchical name information manually in the resource `name` property, by using `/` to format the name.
+
+> **NOTE:** This approach is more error-prone, reduces Bicep's ability to understand relationships between resources as well, and can reduce some of the type validation we are able to perform. We generally recommend instead using either the ['parent property'](#parent-property) or ['nested resource'](#resource-nesting) approach when authoring to avoid these problems.
```

Comment:
Should we mention that we will deprecate this pattern?

Reply:
I'm all for it, but also feel we haven't 100% closed on that decision yet - so didn't want to communicate anything concrete.